pontoon/base/models.py

```diff
@@ -2245,6 +2248,26 @@ def reset_active_translations(self, locale):
 
         translations.filter(pk__in=unreviewed_pks).update(active=True)
 
+    def _get_word_count(self, string):
+        return len(re.findall(r'\w+', string))
+
+    def get_or_create(self, defaults=None, **kwargs):
+        if 'string' in kwargs:
```

Comment:
Why do we need this `if` clause?

Reply:
@mathjazz If there are no 'string' arguments passed to this method - I cannot count words for this entity. Is this situation real? Should I remove this 'if'?

Reply:
Yes, `string` should always be there:
https://github.com/mozilla/pontoon/blob/cc3c3267e8f279f1724d1f79fa0d351cb1c544fb/pontoon/sync/changeset.py#L181